pkg/api/norman/customization/kontainerdriver/actionhandler.go

```diff
@@ -162,12 +162,12 @@ func (lh ListHandler) LinkHandler(apiContext *types.APIContext, next types.Reque
 	var targetImages []string
 	switch apiContext.ID {
 	case linuxImages:
-		targetImages, _, err = image.GetImages(systemCatalogChartPath, "", []string{}, []string{}, rkeSysImages, image.Linux)
+		targetImages, _, err = image.GetImages(systemCatalogChartPath, "", "", []string{}, []string{}, rkeSysImages, image.Linux)
```

Comment:
Need to test this is handled gracefully. It will also not be totally accurate anymore, can't we grab version from inside Rancher ?

Reply:
@PennyScissors Does `settings.ServerVersion.Get()` work here?

Reply:
@kinarashah Nice, I think that should work.

Reply:
We have a couple of these GetServer funcs which use setting and then filter the RC/dev versions as needed by the feature. Could check if we could use any of these or need a new one. 
https://github.com/rancher/rancher/blob/master/pkg/controllers/management/kontainerdrivermetadata/data_getter.go#L208:6 
https://github.com/rancher/rancher/blob/master/pkg/catalog/utils/version.go#L58

Reply:
I'll update this in the next commit